test/helpers/steerable_entity.dart

```diff
@@ -10,4 +11,11 @@ class SteerableEntity extends Entity with Steerable {
 
   @override
   double get maxVelocity => 100;
+
+  @override
+  void renderDebugMode(Canvas canvas) {
+    // Custom debug render mode so that the PositionComponent text doesn't get
+    // rendered.
+    canvas.drawRect(Vector2.zero() & size, debugPaint);
```

Comment:
👀

Reply:
Needed to by pass text failures in golden tests, which we aren't interested anyway